climada/hazard/trop_cyclone.py

```diff
@@ -216,16 +232,20 @@ def set_from_tracks(self, tracks, centroids=None, description='',
                     LOGGER.info("Progress: %d%%", perc)
                     last_perc = perc
                 tc_haz.append(
```

Comment:
I cannot make a suggestion directly for all here, but I suggest to rename `tc_haz` to `tc_haz_list`.

```suggestion
                tc_haz_list.append(
```

Reply:
Fixed in 97b4e31b